lua/telescope/entry_manager.lua

```diff
@@ -26,6 +26,14 @@ if past loop of must have scores,
 local EntryManager = {}
 EntryManager.__index = EntryManager
 
+--- Create a new EntryManager object
+---@param max_results number: the number of results to keep sorted at the head of the linked list
+---@param set_entry function: handler function for object using the EntryManager
+---@param info table: (optional) table containing information to keep track of
+---   @key looped number: number of existing entries checked when adding new entries
+---   @key inserted number: TODO: figure out if this is used anywhere
```

Comment:
I think is currently unused. Tj never did anything with it

Reply:
I think I used it do some debugging for some testing, can probably be removed.